app/touch-bar-menu.js

```diff
@@ -34,6 +34,13 @@ module.exports = function touchBarMenu(window, soundcloud) {
     }
   })
 
+  const repost = new TouchBarButton({
+    icon: `${__dirname}/res/repost.png`,
+    click: () => {
+      soundcloud.repost()
+    }
+  })
+
```

Comment:
The original PR included a reposted icon but no code associated with it. If there was a way to detect if a track has already been reposted we can update the icon, but i'm not sure how to do that.